### PR TITLE
Init react native project

### PR DESCRIPTION
**Pull request by KozlovaAngelika**
Initialized project for react native

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -0,0 +1,66 @@
+[ignore]
```
Comment (brusnev-egor) [inline .flowconfig:1]: remove this file

File: App.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
```
Comment (brusnev-egor) [inline App.tsx:6]: remove comment

File: package.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        "android": "react-native run-android",
+        "ios": "react-native run-ios",
+        "start": "react-native start",
+        "test": "jest",
+        "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+        "prepare": "husky install",
+        "precommit": "lint-staged"
+    },
+    "dependencies": {
+        "prettier-eslint": "^13.0.0",
```
Comment (brusnev-egor) [inline package.json:15]: remove `^` in whole file

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
+    // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
+  },
+  "exclude": [
+    "node_modules", "babel.config.js", "metro.config.js", "jest.config.js"
+  ],
+  "include": [
+    "./**/*",
+    "./*",
+    ".eslintrc.js"
```
Comment (egor-brusnev) [inline tsconfig.json:66]: this file doesn't exists, remove this line

**Review by brusnev-egor — APPROVED** [review]

**Pull request merged**